pyresample/spherical.py

```diff
@@ -112,6 +224,39 @@ def __iter__(self):
         """Get iterator over lon/lat pairs."""
         return zip([self.lon, self.lat]).__iter__()
 
+    def plot(self, ax=None, **plot_kwargs):
+        """Plot the point(s) using Cartopy.
+
+        Assume vertices to be in radians.
+        """
+        import matplotlib.pyplot as plt
+        try:
+            import cartopy.crs as ccrs
+        except ModuleNotFoundError:
+            raise ModuleNotFoundError("Install cartopy to plot spherical geometries.")
+
+        # Create figure if ax not provided
+        ax_not_provided = False
+        if ax is None:
+            ax_not_provided = True
+            proj_crs = ccrs.PlateCarree()
```

Comment:
Not saying this needs to be done now, but should this allow users to provide the CRS? Devil's advocate: What if I don't want cartopy to plot coastlines. I just want to see the spherical geometries plotted together. Why can't I use a matplotlib 3D plot? :wink:

Reply:
I agree with that and we also discussed it with Martin about that. 
If ok for you, we delegate the testing and final design of the plot functions after the new spherical object classes are implemented. 
For the moment, I added the function arguments: `proj_crs=None, add_coastline=True, add_gridlines = True`

Reply:
Sounds good to me.